deploy/kubernetes/setup-csi-snapshotter.yaml

```diff
@@ -110,6 +110,7 @@ spec:
         - name: csi-snapshotter
           image: quay.io/k8scsi/csi-snapshotter:v0.4.0
           args:
+            - "--snapshotter=csi-hostpath"
```

Comment:
This line is not required so we removed it.  It will be detected automatically.  Have you run into any problem without this line?

Reply:
No, just want to keep with provisioner and make the args explicitly